assignments/callbacks.js

```diff
@@ -21,28 +21,56 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 */
 
```

Comment:
good job on all these callback functions. You seem to have gotten to grasp how they work and how to execute on them. Great to see :)